test/e2e/common/empty_dir.go

```diff
@@ -267,7 +267,7 @@ var _ = ginkgo.Describe("[sig-storage] EmptyDir volumes", func() {
 						Name:    busyBoxSubContainerName,
 						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
 						Command: []string{"/bin/sh"},
-						Args:    []string{"-c", fmt.Sprintf("echo %s > %s", message, busyBoxSubVolumeFilePath)},
+						Args:    []string{"-c", fmt.Sprintf("echo %s > %s && sleep 10000", message, busyBoxSubVolumeFilePath)},
```

Comment:
Instead of changing this, can you change CreateSync() to Create() and WaitForRunning (as you did in other tests)/